core/storage/storage_models_test.py

```diff
@@ -25,9 +25,10 @@
 
 (
     base_models, collection_models, email_models,
-    exploration_models, feedback_models, skill_models, topic_models,
-    suggestion_models, user_models, story_models, question_models,
-    config_models) = models.Registry.import_models([
+    exploration_models, feedback_models, skill_models,
+    topic_models, suggestion_models, user_models,
+    story_models, question_models, config_models
+    ) = models.Registry.import_models([
```

Comment:
Deindent by 4 to match indentation of opening paren.

Reply:
Done